client/containers/workflow-list/helpers/get-status.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021 Uber Technologies Inc.
+//
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import { STATUS_LIST, STATUS_LIST_OPTION_DEFAULT } from '../constants';
+
+const getStatus = status =>
+  (status && STATUS_LIST.find(item => item.value === status)) ||
```

Comment:
maybe worth making case-insensitive?  since it's basic-ascii I believe it'll lowercase/uppercase correctly all locales

Reply:
Since status can be entered through the URL I think this is probably a good idea. In practice it probably won't be needed but I can add it.

Reply:
Updated to format status to upper case and added a unit test for this scenario.